rust_snuba/src/strategies/commit_log.rs

```diff
@@ -0,0 +1,90 @@
+use rust_arroyo::backends::kafka::types::KafkaPayload;
+use serde::{Deserialize, Serialize};
+use std::str;
+
+#[derive(Debug, Serialize)]
+struct Commit {
+    topic: String,
+    partition: u16,
+    consumer_group: String,
+    orig_message_ts: f64,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+struct Payload {
+    offset: u64,
+    orig_message_ts: f64,
+}
+
+impl TryFrom<KafkaPayload> for Commit {
+    type Error = anyhow::Error;
```

Comment:
can you define your own error type here?

you can use `thiserror` to derive the necessary traits automatically

I don't think we should use `anyhow` too pervasively in libraries such as rust-arroyo